src/index.js

```diff
@@ -8,4 +8,5 @@ export { default as LayoutHOC } from './layout.hoc'
 export { version } from '../version'
 export const utils = srcUtils
 
+export { default as Col } from './derived/col'
 export { default as Root } from './derived/root'
```

Comment:
This is fine for now but I would like to have a better way to distinguish between core layout components and derived ones